packages/api/lambdas/sf-event-sqs-to-db-records/index.js

```diff
@@ -144,6 +144,8 @@ const writeRecords = async ({
     // then PDR/granules should not be written to Postgres either since they
     // reference executions, so bail out to writing execution/PDR/granule
     // records to Dynamo.
+
+    // TODO - Throw error here in phase 2 instead of writing to Dynamo
```

Comment:
Should this mean Phase 3?  Do we need a JIRA issue?

Reply:
I think it should be Phase 2 -- https://bugs.earthdata.nasa.gov/browse/CUMULUS-2694

Reply:
:thumbsup: